src/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer.c

```diff
@@ -1,11 +1,12 @@
 #include "ac_npc_engineer.h"
 #include "m_lib.h"
 #include "overlays/gamestates/ovl_play/m_play.h"
+#include "include/unknown_structs.h"
 
 void aNEG_actor_ct(Actor* thisx, Game_Play* game_play);
-void aNEG_actor_dt(Actor* thisx, Game_Play* game_play);
-void aNEG_actor_init(Actor* thisx, Game_Play* game_play);
-void aNEG_actor_save(Actor* thisx, Game_Play* game_play);
+void aNEG_actor_dt();
```

Comment:
These are still `ActorFunc` typed so they still take those args even if unused

Reply:
it doesnt match with them there though or at least not with them when they get defined underneath

Reply:
Try passing the missing args to the called function, that may help.
If you have problems don't doubt in asking for help

Reply:
yeah it matched when i did that lmao 
i mean i didnt ask about it cus ig i stumbled upon like a wrong way of doing it that looked right so there didnt seem to have a problem, ill def ask before idk doing a dumb like this again lol